pkg/controller/projectreference/projectreference_controller.go

```diff
@@ -154,15 +166,16 @@ func (r *ReconcileProjectReference) Reconcile(request reconcile.Request) (reconc
 
 		reqLogger.Info(fmt.Sprintf("Setting ProjectReferenceStatus %s", gcpv1alpha1.ProjectReferenceStatusCreating))
 		// passed requirementes check set to creating
-		projectReference.Status.State = gcpv1alpha1.ProjectReferenceStatusCreating
-		err = r.client.Status().Update(context.TODO(), projectReference)
+		adapter.projectReference.Status.State = gcpv1alpha1.ProjectReferenceStatusCreating
+		adapter.projectReference.Status.Conditions = []gcpv1alpha1.ProjectReferenceCondition{}
```

Comment:
Maybe do it similar to the ProjectClaim, where we ensure the condition is initalized once at the beginning? https://github.com/openshift/gcp-project-operator/blob/6e30cdb0d2cf9e9fa03f40e2a09858f6a47f2c5f/pkg/controller/projectclaim/projectclaimadapter.go#L112